aspnetcore/fundamentals/apis.md

```diff
@@ -3,16 +3,21 @@ title: "APIs overview"
 author: tdykstra
 description: Learn about differences between controller-based APIs and minimal APIs.
 ms.author: tdykstra
-ms.date: 01/10/2023
+ms.date: 4/10/2023
+monikerRange: '>= aspnetcore-6.0'
 uid: fundamentals/apis
 ---
 
 # Choose between controller-based APIs and minimal APIs
 
+[!INCLUDE[](~/includes/not-latest-version.md)]
+
 ASP.NET Core supports two approaches to creating APIs: a controller-based approach and minimal APIs. *Controllers* in an API project are classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. *Minimal APIs* define endpoints with logical handlers in lambdas or methods. This article points out differences between the two approaches.
 
 The design of minimal APIs hides the host class by default and focuses on configuration and extensibility via extension methods that take functions as lambda expressions. Controllers are classes that can take dependencies via constructor injection or property injection, and generally follow object-oriented patterns. Minimal APIs support dependency injection through other approaches such as accessing the service provider.
 
+ :::moniker range=">= aspnetcore-7.0"
```

Comment:
To be consistent with what we're doing everywhere else, it might be better to put this at the top of the file under not-latest, even if we don't expect this block of text to change with each version.